### PR TITLE
bj tabbed interface

### PR DESCRIPTION
**Pull request by borvux**

### CODE REVIEW

**Review by reherr — APPROVED** [review]
looks great!

File: app/views/users/_navbar.html.erb
Comment (CarlosCastanedaDev) [inline app/views/users/_navbar.html.erb:1]: Great job making this a partial!

**Review by CarlosCastanedaDev — APPROVED** [review]
Great job, Benny!

You can now merge these changes back to main.

**Comment by CarlosCastanedaDev** [comment]
Please submit this PR link on Canvas so I can grade it and you can move on.

**Pull request merged**